draft-issuance.md

```diff
@@ -0,0 +1,175 @@
+
+“Issuance” - to be interpreted as described in ZIP TBD [3]
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZsfBalanceAfter(h)`” is the total ZEC available in the Zcash Sustainability Fund (ZSF) after the transactions
+in block `h`, described in ZIP #TODO reference#. The Sustainability Fund is used to pay out Block Subsidies
+from unmined ZEC, and other fund deposits.
+
+“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
+
+"`NUMBER_OF_BLOCKS_IN_4_YEARS`" = `(365.25 * 24 * 60 * 60/75 * 4)`, or `1_683_072`
+
+# Abstract
+
+This ZIP proposes a change to how nodes calculate the block subsidy.
+
+Instead of following a step function around the four-year halving cycle inherited
+from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
+scheme would approximate the current 4 year cycle, and results in the last
+zatoshi being issued in around 114 years.
+
+# Motivation
+
+Zcash’s economic model is inherited from Bitcoin and includes the concept of a halving
+mechanism to regulate the issuance of new coins. This approach, though foundational, invites
+ a reevaluation amid Zcash’s ongoing evolution. As the network matures, the need to address
+potential future challenges and ensure a sustained and stable economic ecosystem becomes
+apparent. The transition to a smoothed emissions curve offers an opportunity to adjust the network's
+issuance dynamics while maintaining the supply cap of 21,000,000 coins. By doing so, Zcash
+endeavors to optimize its economic framework, accommodating changing circumstances while
+maintaining predictability and stability in rewards distribution.
+
+This proposal outlines a solution to address challenges associated with the existing block
+subsidy issuance mechanism in the Zcash network. The primary goal of this proposal is to
+introduce a more predictable and stable issuance of ZEC by smoothing out the issuance
+curve while preserving the supply cap. It's important to note that this proposal does
+not seek to alter the fundamental aspects of Zcash's issuance policy. The average block
+subsidy amount over time will remain the same and the funds for block subsidies will last
+a similar amount of time. Instead, it focuses solely on enhancing the predictability
+and consistency of the block subsidy issuance process.
+
+Smoothing the emissions curve helps ensure that the network remains economically
+viable and stable as it transitions from a traditional issuance mechanism to one
+that maintains a sustainable and predictable issuance of rewards over time. It
+prevents abrupt changes in the rate of newly issued coins, which could lead to
+disruptions in the network's economic model and potentially impact its security
+and sustainability. A smoother emissions curve allows for a more gradual and controlled
+transition, providing ZEC stakeholders and participants with a clear understanding of
+how rewards will be distributed over time.
```

Comment:
We think that the key points of these paragraphs are:
- "abrupt changes in the rate of newly issued coins"
- "a [smooth] issuance of ZEC by smoothing out the issuance curve while preserving the supply cap"
- " does not seek to alter the fundamental aspects of Zcash's issuance policy"

We think the motivation will be clearer if the information is presented in this order, and the impact of halvings on the Zcash market is specifically described. To make it easier for the reader to identify these points, we would encourage you to remove verbose wording and buzzwords.